sample/app/http-env-echo-no-loadbalancer.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Right clicking stuff in IDE :/

The issue with `Makefile` path is that the tests are really slow (waiting for cloudformation), the full suite would look like https://github.com/pebble/spacel-provision/blob/master/Makefile#L7 but `test_integ`, but I wouldn't want to wait around for it to run.
You can specify an individual test class/method to `nosetests`; http://nose.readthedocs.io/en/latest/usage.html . An "edit and run me" Makefile might be better than something that tries to run everything.


Bigger picture: Jenkins can/should run the suite nightly?